_episodes_rmd/05-rmarkdown-example.Rmd

```diff
@@ -53,7 +81,7 @@ cat(
 ```
 
 The rest of the lesson should be written as a normal RMarkdown file. You can
-include chunk for codes, just like you'd normally do.
+include chunk for codes, just like you would normally do.
```

Comment:
```suggestion
include chunks of code, just like you would normally do.
```

Reply:
Although we don't want to try to write a complete introduction to RMarkdown in this episode, I think this section would still benefit from a bit more explanation about the output being generated automatically from the code chunks. I've made an attempt at this in #319 so we don't necessarily need to take care of it here.